src/Host/Client/Impl/Definitions/IRCallbacks.cs

```diff
@@ -73,5 +73,8 @@ public interface IRCallbacks {
         /// </summary>
         /// <returns></returns>
         void ViewObject(string expression, string title);
+
+        void PackagesInstalled();
```

Comment:
Hm... ok, let's leave it as it is

---
In reply to: [73555637](https://github.com/Microsoft/RTVS/pull/2149#discussion_r73555637) [](ancestors = 73555637)